requirements.txt

```diff
@@ -1,23 +1,46 @@
 aniso8601==9.0.1
+atomicwrites==1.4.0
 attrs==21.4.0
+black==22.3.0
 certifi==2021.10.8
 charset-normalizer==2.0.12
 click==8.0.3
+colorama==0.4.5
+commonmark==0.9.1
 deepdiff==5.7.0
+flake8==4.0.1
 Flask==2.0.3
 flask-restx==0.5.1
+git+https://github.com/communitiesuk/funding-service-design-utils.git@main
 idna==3.3
+iniconfig==1.1.1
 itsdangerous==2.1.0
 Jinja2==3.0.3
 jsonschema==4.4.0
 MarkupSafe==2.1.0
+mccabe==0.6.1
+mypy-extensions==0.4.3
 ordered-set==4.0.2
+packaging==21.3
+pathspec==0.9.0
+platformdirs==2.5.2
+pluggy==1.0.0
+py==1.11.0
+pycodestyle==2.8.0
+pyflakes==2.4.0
+Pygments==2.12.0
+pyparsing==3.0.9
 pyrsistent==0.18.1
+pytest==7.1.2
 python-dateutil==2.8.2
+python-dotenv==0.20.0
```

Comment:
Where did all these other requirements changes come from? Are they required?

Reply:
@gio-karunakaran any idea?

Reply:
https://digital.dclg.gov.uk/jira/browse/FS-1137 created to cover that work in all repos that need it